core/include/vecmem/containers/const_device_array.hpp

```diff
@@ -11,8 +11,8 @@
 
 namespace vecmem {
 
-   /// Class mimicking a host-filled constant @c std::array in "device code"
-   template< typename T, std::size_t N >
-   using const_device_array = device_array< const T, N >;
+/// Class mimicking a host-filled constant @c std::array in "device code"
+template <typename T, std::size_t N>
+using const_device_array = device_array<const T, N>;
 
-} // namespace vecmem
+}  // namespace vecmem
```

Comment:
Another super subjective one.

I hate the 2 spaces after the namespace closing brackets. But I can just not find in https://clang.llvm.org/docs/ClangFormatStyleOptions.html how to control this. :frowning: (It must be possible, since all the code examples on that page use a single space in such places.) I just really don't see a value in not simply using a single space before `//`, as it was done so far.

Reply:
I think this probably tolerable.